src/components/orders/RowWithCopyButton/index.tsx

```diff
@@ -17,17 +17,18 @@ type Props = {
   textToCopy: string
   contentsToDisplay: string | JSX.Element
   className?: string
+  onCopy?: (value: string) => void
 }
 
 export function RowWithCopyButton(props: Props): JSX.Element {
-  const { textToCopy, contentsToDisplay, className } = props
+  const { textToCopy, contentsToDisplay, className, onCopy } = props
 
   // Wrap contents in a <span> if it's a raw string for proper CSS spacing
   const contentsComponent = typeof contentsToDisplay === 'string' ? <span>{contentsToDisplay}</span> : contentsToDisplay
```

Comment:
this is weird, why not always render the span?

Reply:
Not from this PR, according to the comment is for styling. I agree, it feels a bit strange. 
Maybe @alfetopito wants to add sth.

Reply:
I looks like it can be `string | JSX.Element`, and it's just wrapping the element in case of being a string

I think is just fine...

Reply:
Because:

1. The contents can be a string. In that case, wrap it in a `<span>` for CSS alignment
2. The contents can also be a component. In that case, there's no need to wrap it